about/about-kubernetes-egress.md

```diff
@@ -0,0 +1,111 @@
+---
+title: About Kubernetes egress
+description: Learn about Kubernetes egress!
+---
+
+> <span class="glyphicon glyphicon-info-sign"></span> This guide provides optional background education, including
+> education that is not specific to {{site.prodname}}.
+{: .alert .alert-info}
+
+In this guide you will learn:
+- What is Kubernetes egress?
+- Why should you restrict egress traffic and how can you do it?
+- What is "NAT outgoing" and when is it used?
+- What is an egress gateway, and why might you want to use one?
+
+### What is Kubernetes egress?
+
+In this guide we are using the term Kubernetes egress to describe connections being made from pods to anything outside of the cluster.
+
+In contrast to ingress traffic, where Kubernetes has the [Ingress]({{site.baseurl}}/about/about-kubernetes-ingress)
+resource type to help manage the traffic, there is no Kubernetes Egress resource. Instead, how the egress traffic is
+handled at a networking level is determined by the Kubernetes network implementation / CNI plugin being used by the
+cluster. In addition, if a service mesh is being used, this can add additional egress behaviors on top of what the
+network implementation provides.
```

Comment:
"In addition...this can add additional...on top of..." 

In addition, if a service mesh is being used, this can add egress behaviors on top of what the
network implementation provides.